sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb

```diff
@@ -0,0 +1,1094 @@
+{
```

Comment:
<p>Can you be a little more specific on runtime than "some time," so customers know how long they'll have to wait? 1 minute, 15 minutes, ...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>